feature-libs/customer-ticketing/components/customer-ticketing-list/customer-ticketing-list.component.ts

```diff
@@ -1,11 +1,135 @@
-import { Component, OnInit } from '@angular/core';
+import { Component } from '@angular/core';
+import { of } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { ICON_TYPE } from '@spartacus/storefront';
 
 @Component({
   selector: 'cx-customer-ticketing-list',
   templateUrl: './customer-ticketing-list.component.html',
 })
-export class CustomerTicketingListComponent implements OnInit {
+export class CustomerTicketingListComponent {
+  iconTypes = ICON_TYPE;
+
   constructor() {}
 
-  ngOnInit() {}
+  // Create table Headers
+  tableHeaders = [
+    { key: 'ticketId', label: 'ID' },
+    { key: 'subject', label: 'Subject' },
+    { key: 'ticketCategory', label: 'Category' },
```

Comment:
tableHeader will be removed in a later commit?

Reply:
Not really, they are being used to display the options in the dropdown

Reply:
It should be gone.

An example would be the OrderHistoryComponent. There's no hardcoded things here as the paginated object comes from the backend and we'll be using that to populate the dropdown

Reply:
Yes, this change will be implemented with the next PR that adds backend callls!